packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -1439,7 +1439,8 @@ class _NestedScrollPosition extends ScrollPosition implements ScrollActivityDele
   }
 
   void updateCanDrag(double totalExtent) {
-    context.setCanDrag(totalExtent > (viewportDimension - maxScrollExtent) || minScrollExtent != maxScrollExtent);
+    context.setCanDrag(physics.allowUserScrolling &&
```

Comment:
Fixes #113753

Here seems to be more suitable use `physics.allowUserScrolling`

Reply:
Ah yes, thank you!